### DISCUSSION TITLE
Support SSE (server sent events) - long running Http GET Requests.

### DISCUSSION THREAD

**Discussion by gajanak** [usebruno/bruno#2196] [Ideas]
Hi,

Is there any plan, or workaround for accessing SSE (Browser EventSource) - or mor general Long Polling Http Request ?

I try to work with Timeout - but on Timeout, no response Body is generated.

Does anyone already has a nice solution for this ?

Perfect Solution Would be to Add Some Scripting and provide way: 
Read for N seconds or until 5 Events .... ?

**Comment by crivetimihai** [comment]
Would really love tho see this feature. SSE can return multiple separate JSON objects. curl will just return them as they come.
{ ... }
{ ... }
{ ... }

**Comment by Bilboramix** [comment]
Searched for the feature today. Sad that it's not available...

**Comment by jnsw** [comment]
Any roadmap?

**Comment by nolway** [comment]
That could be very useful!

**Reply by mikhail-barg** [reply]
As far as I can see it does work already. Isn't it?

**Reply by gajanak** [reply]
You are right, in 3.0.2 this features is usable for basic cases.

**Reply by VictorioBerra** [reply]
How do I use it?

**Reply by gajanak** [reply]
> How do I use it?

Simple : Make a Request to a Endpoint using SSE - And You Show the Events instead of a Timeout after long waiting. ;) ?

You can use https://sse.dev  ( https://sse.dev/test ) as a test URL for example.

( The Display of the payload could still be improved. Only HexDump/RAW currently. Json would be fine)
